### PR TITLE
Added the ability to increment a Child's wins count by 1

### PR DESCRIPTION
**Pull request by PLisak777**
# Pull Request Template

Added a simple increment function to the childReducer.js file that will increment the Childs' win count by 1. Added the corresponding action in childActions.js file.
This will also serve as a simple increment function that can be used and deployed across the project anywhere we need a simple counter.

## Description

Add a summary that address the following:

- motivation/purpose for the feature
  To increment a Child's win count by one.
- What were those changes
  Created simple increment reducer function/action to increment Child's win count by 1
- What issues were fixed
  Could not increment Child's win count by 1. Can now Increment Child's win count by 1.
- Provide the link to the user story it is referencing in Trello board or other supporting document
  https://trello.com/c/bfryS6cr

## Commits Checklist

- [x] commits have clear title names
- [x] commit has descriptive message of what has changed
- [x] more small commits than less big commits

## Code Checklist

- [x] code is formatted appropriately
- [x] code meets DRY standards
- [x] no dead code (check for logs and print statements)
- [x] no TODOS instead necessary comments
- [x] code follows Story Squad standards for:
  - Language
  - Framework
  - Libraries

### PRs needs to be reviewed by at least 2 team members

- TPL is the 3rd reviewer
- Lots of comments and suggestions, please!

LOOM HERE!! -- https://www.loom.com/share/86fe0ea8ba0349cbb0ea2ddee324e7b8

[Pull Request Rubric](https://www.notion.so/1fc04e4fedeb429ba873b7c68d281707?v=74054da7991341c0bf970f39410c43da)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -108,6 +108,6 @@
     "jest-environment-jsdom-sixteen": "^1.0.3",
     "jest-prop-type-error": "^1.1.0",
     "lint-staged": "^10.2.11",
-    "prettier-eslint-cli": "^5.0.0"
+    "prettier-eslint-cli": "^4.2.1"
   }
 }
```
Comment (Kyleswillard) [inline package.json:113]: Was this a fresh clone and install?

**Review by Kyleswillard — APPROVED** [review]
Clean and concise. Good work. Noted the package.lock versioning was changed, but other than that looks functional and should work.

**Review by casey-dietz — APPROVED** [review]
Looks good, code is clean and concise and easy to understand. Packages were updated on package.json which may need approval? Nice work though!

File: package.json (hunk 1 above)
Comment (ashtilawat) [reply package.json:113]: You might need to rebase main so that version are the same here.

**Review by ashtilawat — COMMENTED** [review]
Let's rebase main so that the package and package-lock is the same as what is already in our repo. Let's not update version and send the lock up.

**Pull request closed without merging**